plugin/integration_tests/test_service.py

```diff
@@ -26,7 +26,7 @@ class TestService(unittest.TestCase):
     def setUp(self):  # noqa
         self.ctx = MockCloudifyContext()
         current_ctx.set(self.ctx)
-        with open('inputs.yaml') as f:
+        with open('inputs_service.yaml') as f:
```

Comment:
these values will be injected by the testing framework. for now this is ok, I'll email you how to implement the solution fitting the framework.